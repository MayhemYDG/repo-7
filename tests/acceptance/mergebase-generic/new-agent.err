| versions          - semgrep x.y.z on Python x.y.z
| environment       - running in environment git, triggering event is 'unknown'
| manage            - not logged in
=== setting up agent configuration
| using semgrep rules from https://semgrep.dev/c/p/r2c-ci
| reporting findings introduced by these commits:
|   * aaaaaaa add pr finding
| also reporting findings fixed by these commits from the baseline branch:
|   * aaaaaaa fix finding
| to exclude these latter commits, run with
|   --baseline-ref $(git merge-base main HEAD)
| looking at 1 changed path
=== looking for current issues in 1 file
| 1 current issue found
=== not looking at pre-existing issues since all files with current issues are newly created
=== analyzing new issues in this scan
=== exiting with failing status
