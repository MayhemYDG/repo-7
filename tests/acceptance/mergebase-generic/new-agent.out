python.lang.correctness.useless-eqeq.useless-eqeq
     > baz.py:1
     ╷
    1│   15 == 15
     ╵
     = This expression is always True: `15 == 15` or `15 != 15`. If testing for
       floating point NaN, use `math.isnan(15)`, or `cmath.isnan(15)` if the
       number is complex.

